src/gui/application.cpp

```diff
@@ -337,6 +339,19 @@ Application::Application(int &argc, char **argv)
     }
 
     ConfigFile cfg;
+
+    {
+        // these config values will always be empty after the first client run
+        if (!_overrideServerUrl.isEmpty()) {
+             cfg.setOverrideServerUrl(_overrideServerUrl);
+        }
+
+        if (!_overrideLocalDir.isEmpty()) {
+            cfg.setOverrideLocalDir(_overrideLocalDir);
+        }
+    }
```

Comment:
No variable declarations here, so I would say it makes sense to either remove the block and make it inline with the rest of the method, or to put ConfigFile inside the block so that it no longer takes up resources after the block